scripts/kms/jwt_issuer_trust.sh

```diff
@@ -8,6 +8,24 @@ jwt-issuer-trust() {
 
   REPO_ROOT="$(realpath "$(dirname "$(realpath "${BASH_SOURCE[0]}")")/../..")"
 
+  # Check for new issuer
```

Comment:
For my own understanding, the scheme is:

- If you call this without --iss it will use JWT_TOKEN_ISSUER_URL which will have been set if you bring up a local test JWT issuer
- If you have some external JWT issuer you can just specify it with this --iss flag?

If my understanding is correct I really like this scheme!

Reply:
For an external one we need to set JWT_TOKEN_ISSUER_URL  to the token endpoint.
--iss will the name for the iss property in the token. CCF uses this to register the issuer.

Reply:
So is it equivalent to setting `JWT_ISSUER`?  In which case why are we mixing methods? I think we either allow fully specifying which JWT issuer we're trusting via command line params or we just require the env variables to be changed?

Reply:
I added a test adding a new issuer implemented by our test token issuer.
For this case I needed to specify the issuer JWT_ISSUER and keep the url to the test token url.

Reply:
I'm not sure that addresses my point about mixed methods, for the test you could set both env variables for the test, or you could provider command line params for both and specify them for the test?